assets/airports/eddl.json

```diff
@@ -0,0 +1,3131 @@
+                "NVO": ["NVO"]
+            },
+            "draw": [
+                ["DL241", "NVO"]
+            ]
+        },
+        "NVO1U": {
+            "icao": "NVO1U",
+            "name": "Noervenich One Uniform",
+            "rwy": {
+                "EDDL23R": []
+            },
+            "body": ["DL240", "DL241"],
+            "exitPoints": {
+                "NVO": ["NVO"]
+            },
+            "draw": [
+                ["NVO"]
+            ]
+        },
+        "NVO2Y": {
+            "icao": "NVO2Y",
+            "name": "Noervenich Two Yankee",
+            "rwy": {
+                "EDDL05L": []
+            },
+            "body": ["DL050", "DL058", "BAM"],
+            "exitPoints": {
+                "NVO": ["NVO"]
+            },
+            "draw": [
+                ["BAM", "NVO"]
+            ]
+        },
+        "NVO2Z": {
+            "icao": "NVO2Z",
+            "name": "Noervenich Two Zulu",
+            "rwy": {
+                "EDDL05R": []
+            },
+            "body": ["DL050", "DL058", "BAM"],
+            "exitPoints": {
+                "NVO": ["NVO"]
+            },
+            "draw": [
+                ["NVO"]
+            ]
+        },
+        "SONEB3H": {
+            "icao": "SONEB3H",
+            "name": "Soneb Three Hotel",
+            "rwy": {
+                "EDDL23R": []
+            },
+            "body": ["^DL231", "DL234", ["DL246", "S190-"], "DL248", "ERKUM", ["DL245", "A140+"]],
+            "exitPoints": {
+                "SONEB": ["SONEB"]
+            },
+            "draw": [
+                ["DL231", "DL234"],
+                ["ERKUM", "DL245", "SONEB"]
+            ]
+        },
+        "SONEB5J": {
+            "icao": "SONEB5J",
+            "name": "Soneb Five Juliett",
+            "rwy": {
+                "EDDL05L": []
+            },
+            "body": ["DL050", "DL053", "NIKOG", "LUSIX", ["DL054", "A140+"]],
+            "exitPoints": {
+                "SONEB": ["SONEB"]
+            },
+            "draw": [
+                ["DL050", "DL053", "NIKOG", "LUSIX", "DL054", "SONEB"]
+            ]
+        },
+        "SONEB5Z": {
+            "icao": "SONEB5Z",
+            "name": "Soneb Five Zulu",
+            "rwy": {
+                "EDDL05R": []
+            },
+            "body": ["DL050", "DL053", "NIKOG", "LUSIX", ["DL054", "A140+"]],
+            "exitPoints": {
+                "SONEB": ["SONEB"]
+            },
+            "draw": [
+                ["SONEB"]
+            ]
+        },
+        "SONEB7T": {
+            "icao": "SONEB7T",
+            "name": "Soneb Seven Tango",
+            "rwy": {
+                "EDDL23L": []
+            },
+            "body": ["^DL230", ["DL235", "S190-"], "DL248", "ERKUM", ["DL245", "A140+"]],
+            "exitPoints": {
+                "SONEB": ["SONEB"]
+            },
+            "draw": [
+                ["SONEB"]
+            ]
+        }
+    },
+    "stars": {
```

Comment:
STARs should be ordered in some meaningful way. I don't mind if it's alphanumerical or arrivals then CDAs then transitions, whatever as long as it's consistent throughput the file.